packages/block-library/src/search/style.scss

```diff
@@ -0,0 +1,17 @@
+.wp-block-search {
+	display: flex;
+	flex-wrap: wrap;
+
+	.wp-block-search__label {
+		font-weight: bold;
```

Comment:
`font-weight: bold` might be a bit of an opinionated style that themers will need to override. 

Moving it to the editor.scss and theme.scss might be a good idea.

Reply:
Agreed. We should be avoiding opinionated styles as much as possible for core blocks.

Reply:
Moved it to `style.scss` in fb9b79015! 👌